### ISSUE TITLE
isect_tiles function

### ISSUE THREAD

**Issue by djx99** [nerfstudio-project/gsplat#378]
The value of the rendered pixel should be related to the gaussian within 3*sigma, and I don't seem to see a limit on that. With `isect_tiles `function, do all the gaussians in the tile where the overlay pixel is located have an effect on that pixel?
Am I understanding this correctly? Thank you very much

**Comment by liruilong940607** [comment]
Yes. all the gaussians in a tile would affect the pixels living in it. 

The `radii` is calculated using 3*sigma during the projection phase, after that we don't consider 3*sigma anymore.
